src/openInBrowser.js

```diff
@@ -0,0 +1,14 @@
+const opn = require('opn');
```

Comment:
could you move this file to `utils/openBrowser.js`? Thanks!

Reply:
@devongovett sure., updated the PR.